client/src/components/settings/internet.tsx

```diff
@@ -34,6 +34,10 @@ type InternetState = {
   originalValues: Socials;
 };
 
+function Info({ message }: { message: string }) {
+  return <>{message ? <HelpBlock>{message}</HelpBlock> : null}</>;
```

Comment:
```suggestion
  return message ? <HelpBlock>{message}</HelpBlock> : null;
```
Same thing, but less cluttered.

Reply:
🤯